### DISCUSSION TITLE
Bundle size?

### DISCUSSION THREAD

**Discussion by redbar0n** [shadowwalker/next-pwa#290] [Q&A] [answered]
Bundle sizes (minified & gzipped):

next-pwa 5.0.0 took [787.3 kB](https://bundlephobia.com/result?p=next-pwa@5.0.0). 
next-pwa 5.4.0 increased to  [1.4 MB](https://bundlephobia.com/package/next-pwa@5.4.0).

Bundle size was initially a concern, and now even more. It seems prohibitively expensive to require a client to download 1.4 MB, even if it is only once.

But not sure if the bundle size of this library actually matters.

What part of it actually needs to be downloaded by the client, in general?
Will large parts of next-pwa usually be tree-shaken away?

**Comment by shadowwalker** [comment, marked as answer]
Bundle size of this library does not matter at all, since next-pwa is only use in build time and what ships from your server to client browser is workbox library + generated service worker codes.

**Reply by redbar0n** [reply]
Ok great. So how big is the bundle size of that?
